packages/models/project.json

```diff
@@ -13,7 +13,8 @@
         "tsConfig": "packages/models/tsconfig.lib.json",
         "assets": ["packages/models/*.md"],
         "esbuildConfig": "esbuild.config.js"
-      }
+      },
+      "dependsOn": ["^build", "build", "generate-docs"]
```

Comment:
related to your previous comment I added this line... Let me know what you think.